rasa/core/channels/slack.py

```diff
@@ -387,6 +412,35 @@ def get_metadata(self, request: Request) -> Dict[Text, Any]:
 
         return {}
 
+    def is_request_from_slack_authentic(self, request: Request) -> bool:
```

Comment:
Could there be a unit test that can assert this correctly blocks a request with an incorrect signature?